packages/terra-dropdown-button/src/SplitButton.jsx

```diff
@@ -96,13 +137,22 @@ class SplitButton extends React.Component {
   }
 
   handleCaretKeyDown(event) {
-    if (event.keyCode === KeyCode.KEY_SPACE) {
+    if (event.keyCode === KeyCode.KEY_SPACE || event.keyCode === KeyCode.KEY_RETURN) {
       this.setState({ caretIsActive: true });
+      this.setState({ openedViaKeyboard: true });
+    } else if (event.keyCode === KeyCode.KEY_DOWN && this.state.isOpen && !this.state.openedViaKeyboard) {
+      // puts focus on first list element on down arrow key press when dropdown is opened by mouse click
+      const firstOption = this.dropdownList.querySelector('[data-terra-dropdown-first-list-item = "true"]');
```

Comment:
This could be simplified to ⬇️

`this.dropdownList.querySelector('[data-terra-dropdown-first-list-item]');`

Reply:
To keep consistency with attributes had this value here previously.
Simplified to : [this.dropdownList.querySelector('[data-terra-dropdown-first-list-item]')](https://github.com/cerner/terra-core/blob/3d4cbdca1a5c96ae211cdcf5bdb5847ecfdc965c/packages/terra-dropdown-button/src/DropdownButton.jsx#L103)